atlas-engine/ProcessEngine.CompatibilityProxyAdapter
# Fix Overwritten ProcessDefinitions

## Changes

1. This pr fixes the bug, that the deployment of some process definitions _fails_ due to a duplicated definitionId. Since the old ProcessEngine does not use the _definitionId_ at all, the BPMN Studio sets the same value for each ProcessModel.

Since the AtlasEngine internally uses the `DefinitionId`, each deployed Process which was exported with the BpmnStudio will be overwritten. 

To emulate the old behavior, where the `DefinitionId` was not used, the proxy now replaces the old definitionId with the processId, and appends a `_Definition` suffix to it. 

## Issues

Closes #5 

PR: #9

## How to test the changes

> Look at the reproduction guide in the referenced issue and confirm, that now no process silently fails to deploy.
